src/actions/index.js

```diff
@@ -0,0 +1,12 @@
+export const createBook = (book, state) => [
+  ...state,
+  book,
+];
+
+export const removeBook = (book, state) => {
+  const newState = [];
+  state.forEach(({ bookId }) => {
+    if (bookId !== book.bookId) { newState.push(book); }
+  });
+  return newState;
```

Comment:
- [ ] **(OPTIONAL)** Why don't you use the function `filter`? This would make your code shorter :smile:

Reply:
Yeah hehe, thank you